pkg/prove/proof.go

```diff
@@ -84,10 +94,10 @@ func TxInclusion(codec rsmt2d.Codec, data types.Data, txIndex uint64) (types.TxP
 	}, nil
 }
 
-// txSharePosition returns the start and end positions for the shares that
+// TxSharePosition returns the start and end positions for the shares that
 // include a given txIndex. Returns an error if index is greater than the length
 // of txs.
-func txSharePosition(txs types.Txs, txIndex uint64) (startSharePos, endSharePos uint64, err error) {
+func TxSharePosition(txs types.Txs, txIndex uint64) (startSharePos, endSharePos uint64, err error) {
```

Comment:
[optional][can be a FLUP issue] we may consider renaming to `TxSharePositions` because this function returns multiple positions. Alternative naming proposal: `TxShareRange`

Reply:
What does FLUP mean? :D

Reply:
FLUP: Follow-Up

Reply:
Ah,  sure https://github.com/celestiaorg/celestia-app/issues/1241, thanks

Reply:
this func was original done in core before our rather massive refactors to the shares pkg, so just noting mentally that we really need to refactor this one as well. Surprised this works tbh #703